lib/SiftClient.php

```diff
@@ -2,130 +2,350 @@
+        return $this->track('$label', $properties, array(
+            'timeout' => $opts['timeout'],
+            'version' => $opts['version'],
+            'path' => $this->userLabelApiUrl($userId, $opts['version'])
+        ));
     }
 
+
     /**
      * Removes a label from a user
      *
-     * @param $userId A user's id. This id should be the same as the user_id used in event calls.
-     * This parameter is required.
-     * @param $timeout (optional) The number of seconds to wait before failing the request. By default this is
-     * configured to 2 seconds (see above).
+     * @param string $userId  The ID of a user.  This parameter is required.
+     *
+     * @param array $opts  Array of optional parameters for this request:
+     *     - string 'abuse_type': The abuse type for which the user should be unlabeled.
+     *           If omitted, the user is unlabeled for all abuse types.
+     *     - int 'timeout': By default, this client's timeout is used.
+     *     - string 'version': By default, this client's version is used.
+     *
      * @return null|SiftResponse
      */
-    public function unlabel($userId, $timeout = self::DEFAULT_TIMEOUT) {
+    public function unlabel($userId, $opts = array()) {
+        $this->mergeArguments($opts, array(
+            'abuse_type' => null,
+            'timeout' => $this->timeout,
+            'version' => $this->version
+        ));
+
         $this->validateArgument($userId, 'user id', 'string');
 
+        $params = array('api_key' => $this->api_key);
+        if ($opts['abuse_type']) $params['abuse_type'] = $opts['abuse_type'];
+
+        try {
+            $request = new SiftRequest(self::userLabelApiUrl($userId, $opts['version']),
+                SiftRequest::DELETE, $timeout, $version, array('params' => $params));
+            return $request->send();
+        } catch (Exception $e) {
+            return null;
+        }
+    }
+
+
+    /**
+     * Gets the status of a workflow run.
+     *
+     * @param string $run_id  The ID of a workflow run.
+     *
+     * @param array $opts  Array of optional parameters for this request:
+     *     - string 'account_id': by default, this client's account ID is used.
+     *     - int 'timeout': By default, this client's timeout is used.
+     */
+    public function getWorkflowStatus($run_id, $opts = array()) {
+        $this->mergeArguments($opts, array(
+            'account_id' => $this->account_id,
+            'timeout' => $this->timeout
+        ));
+
+        $this->validateArgument($run_id, 'run id', 'string');
+
+        $url = (self::API3_ENDPOINT . '/v3/accounts/'
+                . $opts['account_id'] . '/workflows/runs/' . $run_id);
 
-        $properties = array('api_key' => $this->api_key);
         try {
-            $request = new SiftRequest(self::userLabelApiUrl($userId), SiftRequest::DELETE, $properties, $timeout);
+            $request = new SiftRequest($url, SiftRequest::GET, $opts['timeout'], '3', array(
```

Comment:
This hardcoded '3' looks weird.  Should this be a constant?  (Also for other methods where this occurs.)